web/views/sleep_tracker_view.ex

```diff
@@ -22,10 +22,35 @@ defmodule Healthlocker.SleepTrackerView do
     if Kernel.length(past_week) == 0 do
       0
     else
-      total_slept / Kernel.length(past_week)
+      format_average_sleep(total_slept / Kernel.length(past_week))
     end
   end
 
+  defp hours_mins(humanized_str) do
+    humanized_str
+      |> String.replace(" hours,", "h")
+      |> String.replace(" minutes", "m")
+      |> String.split(",")
+      |> List.first
+  end
+
+  @doc """
+  format_average_sleep
+
+  ## Example
+
+      iex> import Healthlocker.SleepTrackerView
+      iex> format_average_sleep(7.375)
+      "7h 22m"
+
+  """
+
+  def format_average_sleep(num) do
+    Duration.from_hours(num)
+      |> Timex.Format.Duration.Formatter.format(:humanized)
+      |> hours_mins()
```

Comment:
@tonydaly Despite trying over the last couple of hours, I honestly haven't been able to make much of an improvement on this.
DateTimes work fine, but I don't think Durations work well with `strftime`